adapters/sovrn/sovrntest/supplemental/fpd.json

```diff
@@ -0,0 +1,166 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            },
+            {
+              "w": 300,
+              "h": 600
+            }
+          ]
+        },
+        "ext": {
```

Comment:
it kind of seems like support for teh js line 
imp.ext = utils.getBidIdParameter('ext', bid.ortb2Imp) || undefined
is here but it is unclear if it is just unclear, can we change the data passed to something that is not tagid?

Reply:
Yeah, I added another value that's not tagid